catalog/message/catalog.json

```diff
@@ -25,31 +26,27 @@
       "dcat:keyword": [
         "traffic"
       ],
-      "odrl:hasPolicy": [ 
+      "odrl:hasPolicy": [
         {
-          "@context": {
-            "@vocab": "https://www.w3.org/TR/odrl-model/"
-          },
+          "@type": "odrl:Offer",
           "@id": "urn:uuid:2828282:3dd1add8-4d2d-569e-d634-8394a8836a88",
-          "permission": [
+          "odrl:assigner": {
+            "@id": "http://example.com/Provider"
+          },
+          "odrl:permission": [
             {
-              "action": "use",
-              "constraint": [
+              "odrl:action": "odrl:use",
+              "odrl:constraint": [
                 {
-                  "leftOperand": {
-                    "@value": "spatial"
-                  },
-                  "rightOperand": {
-                    "@value": "EU"
+                  "odrl:leftOperand": "odrl:spatial",
+                  "odrl:rightOperand": {
+                    "@id": "_:EU"
                   },
-                  "operator": "EQ"
+                  "odrl:operator": "odrl:eq"
                 }
-              ],
-              "duty": []
+              ]
             }
-          ],
-          "prohibition": [],
-          "obligation": []
+          ]
         }
       ],
       "dcat:distribution": [
```

Comment:
The dct:format should be a reference instead of a plain string. And the dcat:accessService should be a list of DataService instead of a plain string.